quick_start_new.md

```diff
@@ -0,0 +1,29 @@
+## Setup
+Compatible with Python versions 3.8.10 and higher
+### Install dependencies
+```bash
+# Create an virtual environment 
+# (optional but recommended)
+$ python -m venv venv 
+$ source venv/bin/activate
+# install dependencies
+(venv) $ pip install -r requirements_quick_start.txt
+```
+
+## Build Data Profiles
+Config data sources in ver/.dsessions/sources/data_config.yml
```

Comment:
I think this part should utilize the `ver_cli.py` function, namely `create_sources_file`, `add_csv`, and `add_database`. (note: it write the `api_version` and `sources:` twice when I tried it, may need some fixing). Also, I think it's better to git-ignore the session files (?).

Reply:
Fixed the issues with Ver CLI yml writer and add the .dsessions directory into gitignore.